### PR TITLE
Draft: fix shortcuts in menus and more

### PR DESCRIPTION
**Pull request by Roy-043**
To solve the issue of shortcuts not working in menus:
1. New command list format (with strings and tuples) and functions to handle them.
2. There are some duplicate list functions for now. The Arch and BIM workbenches will be updated in separate PRs.

More:
1. Separate list functions for the utilities toolbar and the utilities menu.
2. The order of the utilities toolbar and menu has been improved.
3. The `init_draft_toolbars` and `init_draft_menus` functions have been removed. They were not used anywhere.

As already mentioned this is the first of several PRs. When the Arch and BIM WB have been updated some cleanup will be done in one or more additional PRs.

Forum topic: https://forum.freecadweb.org/viewtopic.php?f=23&t=56804

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [x]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by freecadci** [comment]
FreeCAD CI-repository:
branch:
https://gitlab.com/berndhahnebach/FreeCAD/-/commits/PR_5072

pipeline for commit f4ce6a8291856f:
https://gitlab.com/berndhahnebach/FreeCAD/-/pipelines/378707759

status:
success

all pipelines for each branch: 
https://gitlab.com/berndhahnebach/FreeCAD/-/pipelines?scope=branches

**Comment by berndhahnebach** [comment]
test of a nice freecadci picture

<a href="https://gitlab.com/berndhahnebach/FreeCAD/-/commits/PR_5072">pipeline status</a>

File: src/Mod/Draft/InitGui.py [hunk 1]
```diff
@@ -92,25 +92,47 @@ def QT_TRANSLATE_NOOP(context, text):
 
         # Set up command lists
         import draftutils.init_tools as it
-        self.drawing_commands = it.get_draft_drawing_commands()
+        if hasattr(it, "get_draft_drawing_commands_temp"):
```
Comment (chennes) [inline src/Mod/Draft/InitGui.py:95]: Should there be a note here about removing when BIM is updated?

Comment (Roy-043) [reply src/Mod/Draft/InitGui.py:95]: Actually I was mistaken that this mod affects the BIM workbench. Only the Arch workbench relies on this code. I'll improve the PR with that in mind.

File: src/Mod/Arch/InitGui.py [hunk 2]
```diff
@@ -51,37 +51,83 @@ def QT_TRANSLATE_NOOP(context, text):
         import Draft_rc
         import DraftTools
         import DraftGui
-        from draftguitools import gui_circulararray
```
Comment (Roy-043) [inline src/Mod/Arch/InitGui.py:54]: I have removed the imports from draftguitools. They are not used in the code.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/berndhahnebach/FreeCAD/-/commits/PR_5072">pipeline status</a> for feature branch [PR_5072](https://gitlab.com/berndhahnebach/FreeCAD/-/commits/PR_5072). Pipeline [#381290856 ](https://gitlab.com/berndhahnebach/FreeCAD/-/pipelines/381290856) was triggered at [d6b7ab6](https://github.com/FreeCAD/FreeCAD/pull/5072/commits/d6b7ab69378c421e10d6ab8d6b5df930f60c0156). All CI branch [pipelines](https://gitlab.com/berndhahnebach/FreeCAD/-/pipelines?scope=branches).

**Comment by chennes** [comment]
Is this ready to review now, or are you still working on it?

**Comment by Roy-043** [comment]
This PR is ready for review.

**Comment by yorikvanhavre** [comment]
This looks good to me, but in many of the docstrings the *_temp() functions are still mentioned, although you removed them now. Could you fix that?

**Comment by Roy-043** [comment]
Done.

**Comment by yorikvanhavre** [comment]
Okay, thanks! Merging then!

**Pull request merged**